runtime/bcutil/ConstantPoolMap.hpp

```diff
@@ -287,8 +291,14 @@ class ConstantPoolMap
 	void markMethodRefAsUsedByInvokeSpecial(U_16 methodRefCfrCPIndex)       { mark(methodRefCfrCPIndex, INVOKE_SPECIAL); }
 	void markMethodRefAsUsedByInvokeStatic(U_16 methodRefCfrCPIndex)        { mark(methodRefCfrCPIndex, INVOKE_STATIC); }
 	void markMethodRefAsUsedByInvokeInterface(U_16 methodRefCfrCPIndex)     { mark(methodRefCfrCPIndex, INVOKE_INTERFACE); }
+
+#if defined(J9VM_OPT_OPENJDK_METHODHANDLE)
+	void markMethodRefAsUsedByInvokeHandle(U_16 methodRefCfrCPIndex)        { mark(methodRefCfrCPIndex, INVOKE_HANDLEEXACT); _invokeCacheCount++; }
+	void markMethodRefAsUsedByInvokeHandleGeneric(U_16 methodRefCfrCPIndex) { mark(methodRefCfrCPIndex, INVOKE_HANDLEGENERIC); _invokeCacheCount++;
+#else /* defined(J9VM_OPT_OPENJDK_METHODHANDLE) */
```

Comment:
The original code isn't ifdef-ed under any tag, so I considered the code as default.
The `J9VM_OPT_METHOD_HANDLE` flag is not very consistent through the vm code, I think this is mostly due to method handle need to be supported by default since Java7/8